.env.example

```diff
@@ -9,5 +9,4 @@ TWILIO_AUTH_TOKEN=Your-Twilio-Auth-Token
 TWILIO_NUMBER=Your-Twilio-Phone-Number
 
 # Your mongo db urls
-MONGO_URL=Mongo-Url
```

Comment:
it caused the build to fail: https://travis-ci.org/TwilioDevEd/eta-notifications-node/jobs/177705048